backend/s3/s3.go

```diff
@@ -2717,6 +2764,7 @@ func s3Connection(ctx context.Context, opt *Options, client *http.Client) (*s3.S
 		awsConfig.WithEndpointResolver(r)
 	}
 
+	awsConfig.WithLogLevel(aws.LogDebug)
```

Comment:
I think this isn't supposed to be in this commit.

Maybe we should have an `--s3-debug` flag which turns it on though? Something for a followup PR?

Reply:
You'r right, i removed that line.